.eslintrc

```diff
@@ -32,9 +33,12 @@
     "space-before-function-paren": [2, "never"],
     "space-before-blocks": [2],
     "no-multi-spaces": [2],
+    "no-whitespace-before-property": [2],
```

Comment:
this [one](http://eslint.org/docs/rules/no-whitespace-before-property) may also come handy.